dashboard/modules/metrics/metrics_head.py

```diff
@@ -305,29 +306,29 @@ def _create_default_prometheus_configs(self):
 
     @dashboard_utils.async_loop_forever(METRICS_RECORD_INTERVAL_S)
     async def record_dashboard_metrics(self):
-        dashboard_proc = psutil.Process()
         self._dashboard_head.metrics.metrics_dashboard_cpu.labels(
             ip=self._ip,
             pid=self._pid,
             Component=self._component,
             SessionName=self._session_name,
-        ).set(float(dashboard_proc.cpu_percent()) * 100)
+        ).set(float(self.dashboard_proc.cpu_percent()))
```

Comment:
Is removing `* 100` intentional?

Reply:
yes it's intentional. `cpu_precent` as exported from psutil has already timed 100. It was a bug. The reading in reporter_agent also did not * 100.